test/unit/encode.cpp

```diff
@@ -42,7 +51,7 @@ class encode_test
             opt.space_to_plus =
                 space_to_plus;
             BOOST_TEST(encoded_size(
-                s, opt, test_chars{}) ==
+                s, test_chars{}, opt) ==
```

Comment:
We're moving these again?

https://github.com/boostorg/url/pull/565#discussion_r974825002